src/app/account/login/login.component.ts

```diff
@@ -100,24 +108,26 @@ export class LoginComponent implements OnInit, OnDestroy {
         }
         this.authService.auth(queryParams)
             .takeUntil(this.destroy$)
+            .finally(() => {
+                this.isAuthorizing = false;
+            })
             .subscribe(
-            response => {
-                this.userStoreService.updateUserData(response.data);
-                this.router.navigateByUrl(this.returnURL);
-                this.authState = this.loginStates.LOGGED_IN;
-                this.oAuthCallbackService.removeOAuthEventListener();
-            },
-            error => {
-                if (error.status === 404) {
-                    this.error(LOGIN_ERROR_TYPES.notFound);
-                } else {
-                    this.error(LOGIN_ERROR_TYPES.internalError);
+                response => {
+                    this.userStoreService.updateUserData(response.data);
+                    this.router.navigateByUrl(this.returnURL);
+                    this.authState = this.loginStates.LOGGED_IN;
+                    this.oAuthCallbackService.removeOAuthEventListener();
+                },
+                error => {
+                    if (error.status === 404) {
```

Comment:
do we have 404 in some angular module

Reply:
No, there are no constants for HTTP Status Codes in core Angular modules as of now.